### PR TITLE
add docs for teams

### PR DESCRIPTION
**Pull request by Gankro**

### CODE REVIEW

File: src/doc/crates-io.md [hunk 1]
```diff
@@ -235,3 +219,86 @@ not change over time, and allowing deletion of a version would go against this
...
+publish a crate that you don't own at all, but otherwise happens to have a team.
+
+If you ever change your mind, or just aren't sure if crates.io has sufficient
+permission, you can always go to https://crates.io/login, which will prompt you
+for permission if crates.io doesn't have all the scopes it would like to.
+
+An additional barrier to querying github is that the organization may be
+actively denying third party access. To check this, you can go to:
+
+https://github.com/organizations/:org/settings/oauth_application_policy
```
Comment (alexcrichton) [inline src/doc/crates-io.md]: Should this be indented in to make it a code block?

Comment (Gankro) [inline src/doc/crates-io.md]: I wanted it to be a url, but I guess that makes sense.

**Comment by alexcrichton** [comment]
Could you also tweak the `cargo help owner` page to point to these online docs?

cc @SimonSapin

**Comment by alexcrichton** [comment]
@bors: r+ cfa1ef18098fb1928aefdaec9bbc959aab2af1df

Thanks!

**Pull request merged**